pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -149,8 +154,18 @@ type SecondaryDisks struct {
 }
 
 type OSImage struct {
```

Comment:
please add comments to each structure with explanation of the structutre itself and fields